device/inventec/x86_64-inventec_d7054q28b-r0/pmon.json

```diff
@@ -0,0 +1,5 @@
+{
+    "skip_ledd": true,
+    "skip_xcvrd": true,
```

Comment:
why skip xcvrd?

Reply:
currently deleted the file we no more use it.
the file pmon.json would not take effect until it matches the name as pmon_daemon_control.json.